client/performance-profiler/components/llm-message/style.scss

```diff
@@ -0,0 +1,37 @@
+.performance-profiler-llm-message {
+	box-sizing: border-box;
+	display: flex;
+	padding: 16px;
+	align-items: center;
+	align-self: stretch;
+	justify-content: space-between;
+	background: linear-gradient(0deg, rgba(255, 255, 255, 0.95) 0%, rgba(255, 255, 255, 0.95) 100%), linear-gradient(90deg, #4458e4 0%, #069e08 100%);
+	/* stylelint-disable-next-line scales/radii */
+	border-radius: 6px;
+
+	color: #000;
+	font-family: "SF Pro Text", $sans;
+	font-size: $font-body-small;
+	line-height: 20px;
+	flex-wrap: wrap;
+	gap: 10px;
+
+	.content {
+		display: flex;
+		align-items: center;
+		gap: 8px;
+	}
+
+	.rotate {
+		animation: rotation 3s infinite linear;
```

Comment:
@WBerredo Big Sky has an animated loading icon we can use here. I’ll drop you the file via Slack as GH might over-optimise it.

![ai-loading](https://github.com/user-attachments/assets/9009bcc4-b4fb-4f33-ad75-8e94f6589cc8)